<?xml version="1.0"?>
<launch>    
    <param name="use_sim_time" value="true" />
    
<!--    <node pkg="tf" type="static_transform_publisher" name="tf_base_link_to_torso" args="0 0 0   0 0 0 1 base_link torso 100" /> 
    <node pkg="tf" type="static_transform_publisher" name="tf_head_to_cam" args="0 0 0.1  0.0 0.076 0.0 0.997 Head camera_rgb_frame 100" />  -->
    
    <node name="ite_fusion" pkg="ite_fusion" type="ite_fusion_node"  output="screen" >
        <rosparam file="$(find ite_fusion)/config/slam_params.yaml" command="load"/> 
        
        <param name="odom_input_topic" type="string" value="/itefusion/odom" />
        
        <param name="odom_frame" type="string" value="odom" /> 
        <param name="odom_delay" type="int" value="0" />
        <param name="publish_volume" type="bool" value="false" />
        <param name="publish_points" type="bool" value="true" />
        <param name="publish_points_rate" type="int" value="10" />
        
        
        <param name="base_link_frame" type="string" value="kinect" />  
        <param name="depth_frame" type="string" value="openni_depth_optical_frame" />
        
        <param name="cam_info_topic" type="string" value="/camera/depth/camera_info" />
        <param name="depth_topic" type="string" value="/camera/depth/image" />
        <param name="rgb_topic" type="string" value="/camera/rgb/image_color" />
        
        
    </node>    
    /camera/depth/image_mouse_left
<!--     <node pkg="rosbag" type="play" name="rosbag" required="true" args="- - clock /home/tavu/jetson_ws/bags/icraWS1.bag"/> -->
</launch>
